src/main/java/org/opentripplanner/transit/model/framework/FeedScopedId.java

```diff
@@ -77,6 +77,7 @@ public String getFeedId() {
     return feedId;
   }
 
+  @Nonnull
```

Comment:
Consider removing `@Nonnull`. We should assume that a return value is NONNULL if not annotated with `@nullable`. This apply to about 90% of the returned objects in the code. So, annotating all of these would lead to less readable code - noise. There might be cases where `@Nonnull` is necessary for documentation.

Reply:
Removed.